examples/vault/.test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -ex
```

Comment:
You'll need to run `devenv up&` and wait for the port to be ready

Reply:
Aha, thanks! I now understand the purpose of this test script. I've pushed an updated version.